flow/designs/asap7/aes-block/config.mk

```diff
@@ -30,3 +30,6 @@ export PWR_NETS_VOLTAGES      =
 
 # The macros are very small so use a smaller halo
 export MACRO_PLACE_HALO        ?= 5 5
+
+export CACHED_FLATTEN_POLICY = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/flatten.tcl
+export ADDITIONAL_FILES = $(CACHED_FLATTEN_POLICY)
```

Comment:
Why is this necessary for this one design (along with flatten.tcl)?

Reply:
aes-block is more of an ORFS script test case for blocks and other and not really a quality of results/real design, so it seems like a good choice for testing this feature.